buildSrc/build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+import java.util.*
```

Comment:
```suggestion
import java.util.Properties
```

Reply:
btw I found out why this star-import appeared: the Kotlin official style defines `java.util` as package that should be star-imported.

<img width="859" alt="image" src="https://user-images.githubusercontent.com/897017/223486560-62b24d3c-f26a-4119-ba5d-455cb6cc21fd.png">